src/components/CommentCard/CommentCard.jsx

```diff
@@ -1,16 +1,30 @@
 import React from 'react'
 import './CommentCard.css'
 import moment from 'moment'
+import { deleteArticleComment } from '../../../APIs'
+
+
+export const CommentCard = ({ comment, user, setPostIndicator, postIndicator }) => {
+  const deleteComment = () => {
+
+    deleteArticleComment(comment.comment_id)
```

Comment:
Make sure to include handling for loading states any time an asynchronous request is being made